File: PY/832-flipping-an-image.py
class Solution:
  def flipAndInvertImage(self, A: List[List[int]]) -> List[List[int]]:
    newImg = []
    for cell in A:
      newCell = []
      cell.reverse()
      for num in cell:
        num = 0 if num == 1 else 1
        newCell.append(num)
      newImg.append(newCell)
    return newImg

File: JS/832-flipping-an-image.js
/**
 * @param {number[][]} A
 * @return {number[][]}
 */
var flipAndInvertImage = function(A) {
  let newArr = []
  A.map((x) => {
  let inner = []
  x = x.reverse().map((y) => {
      inner.push(y ? 0 : 1 )
  })
  newArr.push(inner)
})
 return(newArr)
};